### ISSUE TITLE
Track Docker Version in /validate

### ISSUE THREAD

**Issue by brendandburns** [GoogleCloudPlatform/kubernetes#3003]
We require at least docker 1.3.x, the /validate handler should test this, by asking the kubelet what docker version is running.

**Comment by wojtek-t** [comment]
Do you have any preferences how exactly it should look like?
IIUC, currently there is an entry like that for each node:
{
    "component": "node-0",
    "health": "success",
    "msg": "ok",
    "err": "nil"
}

Should this be extended to contain docker version for a given Kubelet (either as part of msg or as an additional field)? Or do you have anything else on your mind? [I'm happy to work on this issue]

**Comment by brendandburns** [comment]
I think that it would be something like:
```json
{
"component": "node-0",
"health": "failure",
"msg": "docker version is too old (1.2.1)",
"err": "nil"
}
```

But I don't feel too strongly.
(and thanks for taking on the issue!)

--brendan

**Comment by brendandburns** [comment]
fixed by #4117
